README.md

```diff
@@ -20,6 +20,7 @@ The Intelligent Kiosk Sample is a collection of demos showcasing workflows and e
 
 | Scenario                     | Overview | Features Covered  |
 | ---------------------------- | -------- | ----------------  |
+| [Anomaly Detector](Documentation/AnomalyDetectorDemo.md)      | A realtime workflow for processing data from a microphone to show point anomalies exported by Microsoft Anomaly Detector Service. | Anomaly Detector API |
```

Comment:
Showcases anomaly detection with three pre-selected data streams, as well as a Live Sound scenario that samples the audio level of your microphone as the input data stream.

Reply:
Fixed